exec/java-exec/src/main/java/org/apache/drill/exec/rpc/data/DataConnectionCreator.java

```diff
@@ -66,8 +67,8 @@ public DataTunnel getTunnel(DrillbitEndpoint endpoint) {
     return new DataTunnel(newManager);
   }
 
-  public void close() {
-    Closeables.closeQuietly(server);
+  public void close() throws IOException {
```

Comment:
No need for `throws IOException`, `Closeables.close(..., true)` makes sure it's not propagated